src/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: #111518;
+}
+
+#navbar {
+  background-color: rgba(238, 191, 51, 0.863);
+  border-radius: 0;
+}
+
+#fakenews{
+  color: rgba(218, 217, 216, 0.863);
+}
+
+#question {
+  color: rgb(255, 200, 35);
+  font-size: 50px;
+}
+
+#category-container {
+  margin: 50px 300px 50px;
+}
+
+#card-container {
```

Comment:
This is referencing a node with an id of `card-container`, but I only see a node with a class of that name. Is this correct? Do you have either css definitions used this way? Am I missing something?

Reply:
I've changed it to articles-container now